javascript/opendcs-web-ui-react/package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "opendcs-web-ui-react",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "api:start": "cd ../../ && ./gradlew runWeb --info",
+    "api:generate": "cd ../../ && ./gradlew :api-client-typescript:build --info",
```

```suggestion
    "api:generate": "./gradlew -p ../../ :api-client-typescript:build --info",
```
Same as above

Side note, does `./gradlew` assume linux? Would this need to be `./gradlew.bat` if running windows?